src/XrdHdfs.cc

```diff
@@ -752,6 +753,7 @@ int XrdHdfsFile::Write(XrdSfsAio *aiop)
 {
 
 // Execute this request in a synchronous fashion
+// Will not be called as XrdHdfsSys::Features() advertises no AIO support.
```

Comment:
You may want to mention pgwrite() since this is the default behavior for that one as well as there is a default implementation in the base class XrdOss.

Reply:
Hmmh, I think it will only confuse people ... it does confuse me :)